From a6f9d67c43d2ebf368add9863c4d4785f48cc498 Mon Sep 17 00:00:00 2001
From: Bharath Vedartham <linux.bhar@gmail.com>
Date: Mon, 12 Aug 2019 10:52:38 -0600
Subject: [PATCH] mmzone: Move boost_watermark definition below struct zone

Move boost_watermark function definition below struct zone to
avoidd compiler-related warnings.

Signed-off-by: Bharath Vedartham <linux.bhar@gmail.com>
---
 include/linux/mmzone.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/mmzone.h b/include/linux/mmzone.h
index ee0ff632d4c0..4f6adf1cf99c 100644
--- a/include/linux/mmzone.h
+++ b/include/linux/mmzone.h
@@ -322,8 +322,6 @@ enum zone_watermarks {
 	NR_WMARK
 };
 
-extern void boost_watermark(struct zone *zone);
-
 #define min_wmark_pages(z) (z->_watermark[WMARK_MIN] + z->watermark_boost)
 #define low_wmark_pages(z) (z->_watermark[WMARK_LOW] + z->watermark_boost)
 #define high_wmark_pages(z) (z->_watermark[WMARK_HIGH] + z->watermark_boost)
@@ -623,6 +621,8 @@ enum zone_flags {
 
 extern int mem_predict(struct frag_info *frag_vec, struct zone *zone);
 
+extern void boost_watermark(struct zone *zone);
+
 static inline unsigned long zone_managed_pages(struct zone *zone)
 {
 	return (unsigned long)atomic_long_read(&zone->managed_pages);
-- 
2.17.1

